{"pytest_version": "7.2.2", "$report_type": "SessionStart"}
{"nodeid": "tests5_64_1.py::test_even_range", "location": ["tests5_64_1.py", 13, "test_even_range"], "keywords": {"test_even_range": 1, "tests5_64_1.py": 1, "Folder_64": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001173200002995145, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "4aacbd5a60e640d784758087981302a9", "node": "<WorkerController gw1>"}
{"nodeid": "tests5_64_1.py::test_large_range", "location": ["tests5_64_1.py", 3, "test_large_range"], "keywords": {"test_large_range": 1, "tests5_64_1.py": 1, "Folder_64": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00014170599979479448, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "4aacbd5a60e640d784758087981302a9", "node": "<WorkerController gw0>"}
{"nodeid": "tests5_64_1.py::test_large_range", "location": ["tests5_64_1.py", 3, "test_large_range"], "keywords": {"test_large_range": 1, "tests5_64_1.py": 1, "Folder_64": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/home/shahin/Benchmark_Command/Q5/command_results_1/Folder_64/generated_answer.py", "lineno": 3, "message": "IndexError: list index out of range"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_large_range():", "        large_list = list(range(0, 4 + 1000))", ">       assert all_neg_ints_inclusive(large_list) == []"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q5/command_results_1/Folder_64/tests5_64_1.py", "lineno": 6, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def all_neg_ints_inclusive(nums):", "        negative_ints = [x for x in range(len(nums)) if nums[x] < 0]", ">       return nums[negative_ints[0]:negative_ints[4]]", "E       IndexError: list index out of range"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q5/command_results_1/Folder_64/generated_answer.py", "lineno": 3, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_large_range():", "        large_list = list(range(0, 4 + 1000))", ">       assert all_neg_ints_inclusive(large_list) == []"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q5/command_results_1/Folder_64/tests5_64_1.py", "lineno": 6, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def all_neg_ints_inclusive(nums):", "        negative_ints = [x for x in range(len(nums)) if nums[x] < 0]", ">       return nums[negative_ints[0]:negative_ints[4]]", "E       IndexError: list index out of range"], "reprfuncargs": {"args": [["nums", "[0, 1, 2, 3, 4, 5, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q5/command_results_1/Folder_64/generated_answer.py", "lineno": 3, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/home/shahin/Benchmark_Command/Q5/command_results_1/Folder_64/generated_answer.py", "lineno": 3, "message": "IndexError: list index out of range"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00016906699966057204, "extra": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "4aacbd5a60e640d784758087981302a9", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
